packages/vuetify/src/composables/group.ts

```diff
@@ -0,0 +1,197 @@
+// Utilities
+import { reactive, provide, inject, computed, onBeforeUnmount } from 'vue'
+import { wrapInArray, uuid, deepEqual } from '@/util/helpers'
+import { consoleWarn } from '@/util/console'
+import { useProxiedModel } from './proxiedModel'
+
+// Types
+import type { Ref, UnwrapRef, InjectionKey, SetupContext } from 'vue'
+
+interface GroupItem {
+  id: string
+  value: Ref<unknown>
+}
+
+interface GroupProps {
+  modelValue?: unknown
+  multiple?: boolean
+  mandatory?: boolean
+  max?: number
+}
+
+interface GroupProvide {
+  register: (item: GroupItem) => void
+  unregister: (id: string) => void
+  toggle: (id: string) => void
+  selected: Ref<any[]>
+  isSelected: (id: string) => boolean
+  prev: () => void
+  next: () => void
+}
+
+export function useGroupItem (
+  props: { value?: unknown },
+  injectKey: InjectionKey<GroupProvide>,
+) {
+  const group = inject(injectKey)
+
+  if (!group) {
+    throw new Error(`Could not find useGroup injection for symbol ${injectKey}`)
+  }
+
+  const id = uuid()
+
+  group.register({
+    id,
+    value: computed(() => props.value),
```

Comment:
```suggestion
    value: toRef(props, 'value'),
```

https://v3.vuejs.org/api/refs-api.html#toref